tools/windows-node-installer/pkg/cloudprovider/azure/vm.go

```diff
@@ -323,6 +323,7 @@ func (az *AzureProvider) createPublicIP(ctx context.Context) (ip *network.Public
 		network.PublicIPAddress{
 			Name:     to.StringPtr(az.IpName),
 			Location: to.StringPtr(nodeLocation),
+			Sku:      &network.PublicIPAddressSku{Name: "Standard"},
```

Comment:
Did you confirm that this is the only place where we need to set the SKU to `standard`?

Please add a link to the Azure docs that mention why SKUs need to match. Do this in the code, commit and PR messages.

Reply:
yes. only place where sku needs to be changed.

Reply:
nit: Make it a constant.